SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -1433,7 +1433,7 @@ void USpatialNetDriver::RemoveActorChannel(Worker_EntityId EntityId)
 {
 	if (!EntityToActorChannel.Contains(EntityId))
 	{
-		UE_LOG(LogSpatialOSNetDriver, Warning, TEXT("RemoveActorChannel: Failed to find entity/channel mapping for entity %lld."), EntityId);
+		UE_LOG(LogSpatialOSNetDriver, Verbose, TEXT("RemoveActorChannel: Failed to find entity/channel mapping for entity %lld."), EntityId);
```

Comment:
What's the legitimate case this actor shows up? There's UNR-709 you should pick up.

Reply:
So I believe the main reason that this happens is that the channel is cleaned up on tearoff. It is then also (attempted to be) cleaned up when the entity is deleted. It's safe in this case to just try and fail to remove the channel.

Reply:
Looks like it" also outputs during a normal authoritative actor deletion, as we clean up the channel and entity locally when we send the entity deletion request. When the remove op comes back then, the actor and channel is already gone."